dependencies.md

```diff
@@ -2,14 +2,16 @@
 
 This file lists the dependencies used in this repository.
 
-| Dependency                                       | License                                 |
-|--------------------------------------------------|-----------------------------------------|
-| github.com/antonmedv/expr                        | MIT License                             |
-| github.com/dustin/go-humanize                    | MIT License                             |
-| github.com/google/go-cmp                         | BSD 3-Clause "New" or "Revised" License |
-| github.com/klauspost/compress                    | MIT License                             |
-| github.com/nats-io/nats-server                   | Apache License 2.0                      |
-| github.com/nats-io/nats.go                       | Apache License 2.0                      |
-| golang.org/x/net                                 | BSD 3-Clause "New" or "Revised" License |
-| golang.org/x/text                                | BSD 3-Clause "New" or "Revised" License |
-| gopkg.in/yaml.v3                                 | Apache License 2.0                      |
+| Dependency                                | License      |
+|-------------------------------------------|--------------|
+| github.com/antonmedv/expr v1.15.3         | MIT          |
+| github.com/dustin/go-humanize v1.0.1      | MIT          |
+| github.com/google/go-cmp/cmp v0.5.9       | BSD-3-Clause |
+| github.com/klauspost/compress v1.17.0     | Apache-2.0   |
```

Comment:
Do we really need versions here? This seems like a recipe for immediatley being out of date as we keep updating dependencies.

Also dont seem worth listing like `compress` and `compress/v2`?

Reply:
Here's an example from a previous cycle PR https://github.com/nats-io/natscli/pull/560/files

No versions, this seems extremely onerous if not strictly needed

Reply:
You're right, versions may be a bit much, I'll remove them.

The thing with `compress` is that `compress/s2` actually has a different license file: https://github.com/klauspost/compress/blob/master/s2/LICENSE

Reply:
ah nice, missed the seperate license, good catch

Reply:
I removed the versions